app/models/ems_event/automate.rb

```diff
@@ -94,7 +94,6 @@ def parse_policy_parameters(target_str, policy_event, param)
       _log.debug("Target object: [#{target.inspect}]")
       _log.debug("Policy source: [#{policy_src}]")
 
-      return if target.nil? || policy_event.nil? || policy_src.nil?
```

Comment:
See 5e1154207d61efb3e5c2f71ecd4f9b8ae6549409 commit message.  Independent of the rest of the PR.